framework/saleor/cart/use-cart.tsx

```diff
@@ -27,17 +27,18 @@ export const handler: SWRHook<
       const data = await fetch({
         ...options,
         variables: {
-          checkoutId: checkoutId,
+          checkoutId: getCheckoutId().checkoutToken,
         },
       })
-      checkout = data.node
+
+      checkout = data;
     }
 
     if (checkout?.completedAt || !checkoutId) {
       checkout = await checkoutCreate(fetch)
     }
 
-    return checkoutToCart({ checkout })
+    return checkoutToCart(checkout);
```

Comment:
Does this work? Or do we have `checkout` inside of `checkout`?

Reply:
@royderks yes, it doesn't return `node` (few lines above)